### PR TITLE
Hero section redesign

### PR DESCRIPTION
**Pull request by m-abdelwahab**
# Before

image

# After

image

# Changes

1. Change button border-radius
2. Updated GH CTA
3. Navbar resizing
4. Code Switcher that adds context when hovering
5. Change title casing for readability
6. Add logos section along with links to case studies

### CODE REVIEW

File: components/Companies.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              className="w-32 h-32 md:w-36 lg:w-44 md:h-36 lg:h-44"
+              src={site.src}
+              key={site.src}
+              alt={site.alt}
+            />
+          </a>
+        ))}
+      </div>
+      <p className="mt-5 text-xl text-center">
+        <span className="opacity-75">Check out case studies from</span>{' '}
```
Comment (sw-yx) [inline components/Companies.js:45]: technically the rest of these are not case studies

```suggestion
        <span className="opacity-75">Check out more from</span>{' '}
```

File: components/Companies.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <div>
+      <p className="text-center text-2xl mt-10 mb-3 capitalize font-bold tracking-wide">
+        Trusted in production by
+      </p>
+      <div className="flex flex-wrap items-center justify-center space-x-5 font-bold ">
+        {sites.map((site, i) => (
+          <a key={i} className="" href={site.url}>
+            <img
+              className="w-32 h-32 md:w-36 lg:w-44 md:h-36 lg:h-44"
```
Comment (sw-yx) [inline components/Companies.js:36]: can we do something on hover? this is an svg so we can color it in with `currentColor`

suggestion

```suggestion
          <a key={i} className="hover:text-temporalblue" href={site.url}>
```

File: components/Companies.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    alt: 'Descript logo',
+    url: 'https://docs.temporal.io/blog/descript-case-study'
+  }
+];
+
+export default function Companies() {
+  return (
+    <div>
+      <p className="text-center text-2xl mt-10 mb-3 capitalize font-bold tracking-wide">
+        Trusted in production by
```
Comment (sw-yx) [inline components/Companies.js:30]:
```suggestion
        Trusted by
```

File: components/Companies.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    src: '/logos/logo-Box.svg',
+    alt: 'Box logo',
+    url: 'https://docs.temporal.io/blog/Temporal-a-central-brain-for-Box'
+  },
+  {
+    src: '/logos/logo-Descript.svg',
+    alt: 'Descript logo',
+    url: 'https://docs.temporal.io/blog/descript-case-study'
+  }
+];
```
Comment (sw-yx) [inline components/Companies.js:24]: can you also add zebra medical vision and airbyte logos pls

```suggestion
  {
    src: '/logos/logo-Descript.svg',
    alt: 'Descript logo',
    url: 'https://docs.temporal.io/blog/descript-case-study'
  },
  {
    src: '/logos/logo-Zebra.svg',
    alt: 'Zebra logo',
    url: 'https://docs.temporal.io/blog/zebra-medical-case-study'
  },
  {
    src: '/logos/logo-Airbyte.svg',
    alt: 'Airbyte logo',
    url: 'https://docs.temporal.io/blog/airbyte-case-study'
  }
];
```

File: components/Hero.js [hunk 5]
```diff
@@ -1,117 +1,51 @@
...
+            <img className="w-5 h-5" src="/icons/icon-github.svg" alt="GitHub Icon" />
+            <p>
+              <a href="https://github.com/temporalio/temporal" rel="noreferrer" target="_blank">
+                Check out Temporal on GitHub
+              </a>
             </p>
+            <div className="flex space-x-2 items-center text-lg">
+              <img className="w-5 h-5" src="/icons/icon-star.svg" alt="GitHub Icon" />
+              <p className="font-medium">2210</p>
+            </div>
```
Comment (sw-yx) [inline components/Hero.js:38]: Suggestion

```suggestion
            <p>
              <a href="https://github.com/temporalio/temporal" rel="noreferrer" target="_blank">
                Open Source + Self Hostable
              </a>
            </p>
```

File: components/Hero.js [hunk 6]
```diff
@@ -1,117 +1,51 @@
...
+        <div className="my-12 md:my-14 flex flex-col space-y-5 justify-center items-center">
+          <h1 className="text-60 leading-60 lg:text-8xl tracking-wide capitalize font-bold text-center">
+            Build Invincible Apps
+          </h1>
+          <p className="text-xl md:text-2xl max-w-screen-md text-center">
+            Temporal is the open source runtime for running highly reliable, scalable microservice
+            orchestration for mission-critical applications.
+          </p>
+          <div className="py-4 text-lg font-medium flex flex-col justify-center sm:justify-start space-y-4 sm:space-y-0 sm:flex-row sm:space-x-4 items-center">
+            <DirectionalControl href="#explain-temporal">Get Started</DirectionalControl>
```
Comment (sw-yx) [inline components/Hero.js:23]:
```suggestion
            <DirectionalControl href="#explain-temporal">2 Minute Video</DirectionalControl>
```

File: components/Hero.js [hunk 7]
```diff
@@ -1,117 +1,51 @@
...
+            <DirectionalControl secondary={true} href="#explain-temporal">
+              How it works
```
Comment (sw-yx) [inline components/Hero.js:25]: same link why

**Review by sw-yx — CHANGES_REQUESTED** [review]

File: components/Hero.js (hunk 7 above)
Comment (m-abdelwahab) [reply components/Hero.js:25]: typo

**Comment by feedmeapples** [comment]
the three window dots.. they are quite bright and stand out from the rest of the page when don't seem to have a purpose?

image

**Comment by m-abdelwahab** [comment]
> the three window dots.. they are quite bright and stand out from the rest of the page when don't seem to have a purpose?
> 
> image

@feedmeapples so they're supposed to make the tab component look more like a window, similar to VSCode. For the colors, I can definitely pick a darker color

**Comment by feedmeapples** [comment]
> @feedmeapples so they're supposed to make the tab component look more like a window, similar to VSCode. For the colors, I can definitely pick a darker color

i do like the lines indexes and lang tab buttons a lot! Still not sure about the pseudo-buttons even if they are darker. Code-block components are quite common and imho windowish enough while minimal.
Then there is also extra blank space. Imho if the tab buttons were at the top of the code block would look super slick
image

**Comment by feedmeapples** [comment]
Looks good!

File: components/CodeSwitcher/CodeSwitcher.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+          <span className="h-3 w-3 bg-[#f56565] rounded-full"></span>{' '}
+          <span className="ml-2 h-3 w-3 bg-[#fbd38d] rounded-full"></span>{' '}
+          <span className="ml-2 h-3 w-3 bg-[#48bb78] rounded-full"></span>
+        </div>
+        <TabList className="text-white drop-shadow-lg rounded-tr-lg rounded-tl-lg p-2 ml-2 flex cursor-pointer space-x-5 overflow-auto">
+          <Tab selectedClassName="bg-temporalblue text-spaceblack p-1 rounded-lg">
+            <p>Go</p>
+          </Tab>
+          <Tab selectedClassName="bg-green1 text-spaceblack p-1 rounded-lg">
+            <p>Node</p>
```
Comment (sw-yx) [inline components/CodeSwitcher/CodeSwitcher.js:24]: we are rebranding to TypeScript

```suggestion
            <p>TypeScript</p>
```

can you make sure to change in the rest of the code

File: components/Companies.js (hunk 4 above)
Comment (m-abdelwahab) [reply components/Companies.js:24]: Sure! 👍🏻

File: components/CodeSwitcher/CodeSwitcher.js (hunk 8 above)
Comment (m-abdelwahab) [reply components/CodeSwitcher/CodeSwitcher.js:24]: Rebranding to TS makes a lot of sense

File: components/Companies.js [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+        ,{' '}
+        <a
+          className="nav-link"
+          href="https://www.youtube.com/watch?v=kDlrM6sgk2k&t=1188s&ab_channel=HashiCorp">
+          Hashicorp
+        </a>{' '}
+        &{' '}
+        <a className="nav-link" href="https://docs.temporal.io/blog/tags/case-study">
+          more
+        </a>
```
Comment (sw-yx) [inline components/Companies.js:76]:
```suggestion
        <span className="opacity-75">Check out more from</span>{' '}
        <a
          className="nav-link"
          href="https://eng.snap.com/build_a_reliable_system_in_a_microservices_world_at_snap/">
          Snap
        </a>
        ,{' '}
        <a
          className="nav-link"
          href="https://www.youtube.com/watch?v=kDlrM6sgk2k&t=1188s&ab_channel=HashiCorp">
          Hashicorp
        </a>
        ,{' '}
        <a
          className="nav-link"
          href="https://www.youtube.com/watch?v=eWFpl-nzGsY&feature=youtu.be">
          Datadog
        </a>
        ,{' '}
        <a
          className="nav-link"
          href="https://doordash.engineering/2020/08/14/workflows-cadence-event-driven-processing/">
          Doordash
        </a>{' '}
        &{' '}
        <a className="nav-link" href="https://docs.temporal.io/docs/external-resources">
          others
        </a>
```

File: components/Hero.js [hunk 10]
```diff
@@ -1,117 +1,51 @@
...
+          <div className="flex space-x-2 items-center md:text-lg ">
+            <img className="w-5 h-5" src="/icons/icon-github.svg" alt="GitHub Icon" />
+            <p>
+              <a href="https://github.com/temporalio/temporal" rel="noreferrer" target="_blank">
+                Check out Temporal on GitHub
+              </a>
             </p>
+            <div className="flex space-x-2 items-center text-lg">
+              <img className="w-5 h-5" src="/icons/icon-star.svg" alt="GitHub Icon" />
+              <p className="font-medium">2210</p>
```
Comment (sw-yx) [inline components/Hero.js:37]:
```suggestion
```

File: components/Companies.js [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+  return (
+    <div>
+      <p className="text-center text-2xl mt-10 mb-3 capitalize font-bold tracking-wide">
+        Trusted by
+      </p>
+      <div className="flex flex-wrap md:space-x-5 font-bold items-center justify-center">
+        {sites.map((site, i) => (
+          <a className="md:mt-4 flex" key={i} href={site.url}>
+            <img
+              className="w-32 h-32 md:w-44 md:h-44"
```
Comment (sw-yx) [inline components/Companies.js:46]:
```suggestion
              className="w-32 h-32 md:w-44 md:h-44 hover:text-temporalblue"
```

File: public/logos/logo-Zebra-Medical-Vision.svg [hunk 12]
```diff
@@ -0,0 +1,11 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Zebra-Medical-Vision.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

File: public/logos/logo-Descript.svg [hunk 13]
```diff
@@ -0,0 +1,8 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Descript.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

File: public/logos/logo-Coinbase.svg [hunk 14]
```diff
@@ -1,4 +1,4 @@
-<svg width="1024" height="1024" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
-<rect width="1024" height="1024" rx="512" fill="#0052FF"/>
-<path d="M512.147 692C412.697 692 332.146 611.45 332.146 512C332.146 412.55 412.697 332 512.147 332C601.247 332 675.197 396.95 689.447 482H870.797C855.497 297.2 700.846 152 512.147 152C313.396 152 152.146 313.25 152.146 512C152.146 710.75 313.396 872 512.147 872C700.846 872 855.497 726.8 870.797 542H689.297C675.047 627.05 601.247 692 512.147 692Z" fill="white"/>
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Coinbase.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

File: public/logos/logo-Checkr.svg [hunk 15]
```diff
@@ -0,0 +1,9 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Checkr.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

File: public/logos/logo-Box.svg [hunk 16]
```diff
@@ -0,0 +1,5 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Box.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

File: public/logos/logo-Airbyte.svg [hunk 17]
```diff
@@ -0,0 +1,5 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none"
```
Comment (sw-yx) [inline public/logos/logo-Airbyte.svg:1]:
```suggestion
<svg width="300" height="300" viewBox="0 0 300 300" fill="currentColor"
```

**Pull request merged**